### ISSUE TITLE
compilation failure with ghc 9.0.2

### ISSUE THREAD

**Issue by bergmark** [agda/hs-tags#6]
```
Building executable 'hs-tags' for hs-tags-0.1.5.2..
[3 of 3] Compiling Main             ( Main.hs, /Users/adam.bergmark/repos/3rd/unpacks/hs-tags-0.1.5.2/dist-newstyle/build/x86_64-osx/ghc-9.0.2/hs-tags-0.1.5.2/x/hs-tags/noopt/build/hs-tags/hs-tags-tmp/Main.o )

Main.hs:240:40: error:
    • The constructor ‘IncludeSpecs’ should have 3 arguments, but has been given 2
    • In the pattern: IncludeSpecs qs gs
      In a case alternative:
          IncludeSpecs qs gs -> IncludeSpecs qs (optIncludePath opts ++ gs)
      In the ‘includePaths’ field of a record
    |
240 |                                        IncludeSpecs qs gs -> IncludeSpecs qs (optIncludePath opts ++ gs)
    |                                        ^^^^^^^^^^^^^^^^^^
```

**Comment by andreasabel** [comment]
Thanks for the report @bergmark !  
Seems like the GHC API changed from 9.0.1 to 9.0.2.  I could not find any documentation about this change short of doing a git-blame on the ghc sources.
The fix is easy, though...

**Comment by andreasabel** [comment]
Fix released in 0.1.5.3
